src/HomogeneousCalorimeter_geo.cpp

```diff
@@ -196,33 +208,81 @@ static Ref_t create_detector(Detector& desc, xml::Handle_t handle, SensitiveDete
 // helper function to build module with or w/o wrapper
 std::tuple<Volume, Position> build_module(Detector& desc, xml::Collection_t& plm, SensitiveDetector& sens)
```

Comment:
```suggestion
std::tuple<Volume, Position> build_module(Detector& desc, xml::Collection_t& plm, SensitiveDetector& sens)
```

Reply:
I change this in my local. Should I commit and push again?

Reply:
No, I applied it to get this to move forward.